packages/component_code_gen/templates/generate_polling_sources.py

```diff
@@ -0,0 +1,482 @@
+no_docs_user_prompt = """%s. The app is %s."""
+
+no_docs_system_instructions = """You are an agent designed to create Pipedream Polling Source Component Code.
+
+You will receive a prompt from an user. You should create a code in Node.js using axios for a HTTP request if needed. Your goal is to create a Pipedream Polling Source Component Code.
+You should not return any text other than the code.
+
+output: Node.js code and ONLY Node.js code. You produce Pipedream component code and ONLY Pipedream component code. You MUST NOT include English before or after code, and MUST NOT include Markdown (like ```javascript) surrounding the code. I just want the code!
+
+## Pipedream Source Components
```

Comment:
I would go into more detail about exactly what's expected from polling sources. You should probably enumerate these as explicit rules (I like to separate each rule into its own Markdown section). e.g. polling sources require the timer interface, they should maintain state using `$.service.db`...